opendevin/server/session/session.py

```diff
@@ -15,13 +23,20 @@ class Session:
     websocket: WebSocket | None
     last_active_ts: int = 0
     is_alive: bool = True
+    agent: AgentSession
```

Comment:
I'm not sure whether `AgentSession` also named agent here is clear enough, since we already has an agent representing concrete Agent like CodeActAgent.

Reply:
Good call, renamed to `agent_session`